frontend/cypress/integration/common/table.ts

```diff
@@ -76,13 +76,12 @@ And('user filters for istio type {string}', (istioType: string) => {
 });
 
 // checkCol
-// This func assumes:
-//
-// 1. There is only 1 table on the screen.
-//
-// Be aware of these assumptions when using this func.
-export function colExists(colName: string, exists: boolean) {
-  return cy.get(`th[data-label="${colName}"]`).should(exists ? 'exist' : 'not.exist');
+export function colExists(colName: string, exists: boolean, numberOfTables?:number) {
+  if (numberOfTables === undefined){
+    numberOfTables = 1;
+  }
+  return cy.get(`th[data-label="${colName}"]`).should(exists ? 'exist' : 'not.exist')
+  .and('have.length',`${numberOfTables}`);
```

Comment:
This is just a guess but it's possible that adding the `and` here is what broke the tests. I think having that `and` will cause the test to look for the col in the DOM regardless of whether or not it should exist. Try removing the `and` then seeing if the tests pass. Really there should probably be two functions here `colExists` and `colNotExists` rather than trying to combine the functionality into a single one.

Reply:
I decided to remove the implementation from this PR.